fe1-web/parts/lao/organizer/RollCallOpened.tsx

```diff
@@ -69,6 +77,25 @@ const RollCallOpened = () => {
     }
   };
 
+  const handleEnterManually = (input: string) => {
+    if (base64Matcher.test(input)) {
+      if (!attendees.has(input)) {
+        updateAttendees((prev) => new Set<string>(prev.add(input)));
+        toast.show(STRINGS.roll_call_participant_added, {
```

Comment:
Since the toast is the same as handleScan, you can factorize them into a single function.

Reply:
Actually it's not the same toast, the text is not the same. I'll try to find a way to have less duplication though.